aiohttp_wsgi/wsgi.py

```diff
@@ -239,7 +238,13 @@ def _get_environ(self, request, body, content_length):
     async def handle_request(self, request):
         # Check for body size overflow.
         if request.content_length is not None and request.content_length > self._max_request_body_size:
-            raise HTTPRequestEntityTooLarge()
+            try:
+                raise HTTPRequestEntityTooLarge(
+                    max_size=self._max_request_body_size,
+                    actual_size=request.content_length,
+                )
```

Comment:
I don't think that shimming multiple versions of aiohttp is a particularly scaleable way to go. I'd prefer to simply raise the minimum version in setup.py to 3.4, and avoid the branching code paths.

Reply:
Ok, I'll update the PR.